docs/snippets/prerequisites.md

```diff
@@ -2,17 +2,17 @@
 
 Before installing Knative, you must meet the following prerequisites:
 
-- **For prototyping purposes**, Knative will work on most local deployments of Kubernetes. For example, you can use a local, one-node cluster that has 2 CPU and 4GB of memory.
+- **For prototyping purposes**, Knative works on most local deployments of Kubernetes. For example, you can use a local, one-node cluster that has 2&nbsp;CPUs and 4&nbsp;GB of memory.
 
     !!! tip
         You can install a local distribution of Knative for development use by following the [Getting started guide](../../../../getting-started/){_blank}.
 
 - **For production purposes**, it is recommended that:
-    - If you have only one node in your cluster, you will need at least 6 CPUs, 6 GB of memory, and 30 GB of disk storage.
-    - If you have multiple nodes in your cluster, for each node you will need at least 2 CPUs, 4 GB of memory, and 20 GB of disk storage.
+    - If you have only one node in your cluster, you need at least 6&nbsp;CPUs, 6&nbsp;GB of memory, and 30&nbsp;GB of disk storage.
+    - If you have multiple nodes in your cluster, for each node you need at least 2&nbsp;CPUs, 4&nbsp;GB of memory, and 20&nbsp;GB of disk storage.
```

Comment:
I'm not sure why this has `&nbsp;`? Is this intentional?

Reply:
Yes, it's in accordance with [Google Style](https://developers.google.com/style/units-of-measure#spaces-in-units-of-measurement).

Reply:
If we follow some Style from other Style Guides, let's add it to our current Style Guide as we go please?